pkgs/os-specific/linux/minimal-bootstrap/stage0-posix/default.nix

```diff
@@ -1,9 +1,13 @@
 { lib
 , newScope
+, make-minimal-bootstrap-sources
 }:
 
 lib.makeScope newScope (self: with self; {
-  inherit (import ./bootstrap-sources.nix) version hex0-seed src;
+  inherit (import ./bootstrap-sources.nix { inherit make-minimal-bootstrap-sources; })
```

Comment:
I suppose if we remove `make-minimal-bootstrap-sources` from above as @emilytrau suggests then we need to use `self.callPackage` here, right? Sounds good to me.

Reply:
52edaab39be9